test/parallel/test-http-set-trailers.js

```diff
@@ -103,8 +101,8 @@ server.on('listening', function() {
     headers: {}
   }, function(res) {
     res.on('end', function() {
-      //console.log(res.trailers);
-      assert.ok('x-foo' in res.trailers, 'Client doesn\'t see trailers.');
+      assert.ok('x-foo' in res.trailers,
+                'Client doesn\'t see trailers in res.trailers');
```

Comment:
`res.trailers` is an Object. Should we use the `Object,keys` to show all the keys? @BridgeAR

Reply:
Maybe just use `util.inspect`? `Object.keys` works just as fine as well though, as it is only about the keys in this case.